### DISCUSSION TITLE
Make sound waveform and labels move synchronously with insert and delete sound

### DISCUSSION THREAD

**Discussion by troglobytor** [audacity/audacity#2857] [Q&A] [answered]
Unless I missed how such is already a feature, make sound waveform and labels move synchronously with insert and delete sound segment.

**Comment by petersampsonaudacity** [comment, marked as answer]
@troglobytor  see:  https://manual.audacityteam.org/man/sync_locked_track_groups.html

**Comment by troglobytor** [comment]
I'm new to Audacity and I did not copy labels with soundtrack and paste into another track, but only soundtrack was copied which caused labels to be off in the new track. Please forget about suggestion.
